libevm/JitVM.cpp

```diff
@@ -175,9 +174,27 @@ void create(evm_result* o_result, ExtVMFace& _env, evm_message const* _msg) noex
 	}
 	else
 	{
-		o_result->code = EVM_FAILURE;
-		o_result->output_data = nullptr;
-		o_result->output_size = 0;
+		o_result->code = EVM_REVERT;
```

Comment:
@gumb0 @axic Is my explanation enough at this moment or should I address the issue first?